htdocs/class/xoopsform/formbutton.php

```diff
@@ -111,6 +111,10 @@ public function getType()
      */
     public function render()
     {
-        return "<input type='" . $this->getType() . "' class='formButton' name='" . $this->getName() . "'  id='" . $this->getName() . "' value='" . $this->getValue() . "' title='" . $this->getValue() . "'" . $this->getExtra() . ' />';
+        if ($GLOBALS['xoopsConfig']['bootstrap'] == true){
+            return "<input type='" . $this->getType() . "' class='btn btn-default' name='" . $this->getName() . "'  id='" . $this->getName() . "' value='" . $this->getValue() . "' title='" . $this->getValue() . "'" . $this->getExtra() . ' />';
+        } else {
```

Comment:
__[phpmd]__
* The method render uses an else expression. Else is never necessary and you can simplify the code to work without else.

<!-- Comment by [SideCI](https://sideci.com) -->